Dockerfile

```diff
@@ -1,19 +1,19 @@
-FROM amazoncorretto:11 AS builder
+FROM amazoncorretto:17 AS builder
 ARG SPRING_MONGO_URL
 ARG PUBLIC_KEY
 ARG KAFKA_BOOTSTRAP_SERVERS
 WORKDIR /app
 COPY gradle gradle
 COPY src src
 COPY gradlew gradlew
-COPY build.gradle build.gradle
-COPY settings.gradle settings.gradle
+COPY build.gradle.kts build.gradle.kts
+COPY settings.gradle.kts settings.gradle
```

Comment:
This instruction should be
`COPY settings.gradle.kts settings.gradle.kts`

Reply:
fixed